pkg/asset/asset.go

```diff
@@ -56,6 +56,9 @@ const (
 	AssetPathSystemNamespace                = "manifests/kube-system-ns.yaml"
 	AssetPathCheckpointer                   = "manifests/pod-checkpointer.yaml"
 	AssetPathEtcdOperator                   = "manifests/etcd-operator.yaml"
+	AssetPathSelfHostedEtcdMemberPeerSecret = "manifests/etcd-member-peer-tls.yaml"
+	AssetPathSelfHostedEtcdMemberCliSecret  = "manifests/etcd-member-client-tls.yaml"
+	AssetPathSelfHostedOperatorEtcdSecret   = "manifests/operator-etcd-client-tls.yaml"
```

Comment:
Can you clarify the intended use of the MemberPeer, MemberCli, and Operator certificates? This came up previously when client certs were almost used between etcd peers.